example/material.json

```diff
@@ -64,6 +64,14 @@
             "element": "Si",
             "name": "elemental_ratio",
             "value": 0.71
+        },
+        {
+            "name": "inchi",
+            "value": "Not Available"
+        },
+        {
+            "name": "inchi_key",
+            "value": "Not Available"
```

Comment:
For materials this can be set to "Not Available", "Null", etc. For molecules this would be an actual value.

Reply:
let's use the appropriate null value then: https://stackoverflow.com/questions/21120999/representing-null-in-json